src/main/java/com/alkemy/ong/data/gateways/DefaultCommentaryGateway.java

```diff
@@ -47,6 +49,34 @@ public Commentary create(Commentary commentary) {
                 .collect(toList());
     }
 
+    @Override
+    public Commentary findById(Long id) {
+        CommentaryEntity comm= commentaryRepository.findById(id).orElseThrow(() -> new BadRequestException("mensaje"));
```

Comment:
pongamos un mensaje acorde.. 
Ademas.. para el not found nos conviene utilizar la excepción ResourceNotFound q ya la mapeamos para q retorne el 404 en el global exception handler!